Gaby-Martinez/appointments-back
# [AP-16]: configure backend cd

## Description

- Added CD to run in self hosted runner:
  - Modified CI to run with CD
  - Configure Minikube
  - Build Image with Docker
  - Add Kubernetes Manifest
  - Add trigger for tags

## Type of change


## How Has This Been Tested?

- Create tags and verified that the actions have run correctly:
![imagen](https://github.com/user-attachments/assets/598ce18e-3171-41f6-8b1f-b6e47c6a793a)
- Verify that it's available locally with kubectl port-forward
![imagen](https://github.com/user-attachments/assets/22334871-0b60-4254-a793-e5e011d7754e)


## Checklist:
